core/src/main/java/com/griefcraft/bukkit/NMS.java

```diff
@@ -0,0 +1,23 @@
+package com.griefcraft.bukkit;
+
+import org.bukkit.World;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Entity;
+
+public abstract interface NMS extends Block {
+
```

Comment:
What is the purpose of this interface?

Reply:
Technically not required - just was a leftover piece of the code I borrowed from the other project. I can pull that out if you think it adds too much. It's not implemented anywhere else, since I didn't use all of the wrappers that they did.

Reply:
Yeah I'd rather not add extraneous code

Reply:
Fixed! :)